.buildkite/run-cpu-test.sh

```diff
@@ -10,5 +10,22 @@ remove_docker_container() { docker rm -f cpu-test || true; }
 trap remove_docker_container EXIT
 remove_docker_container
 
-# Run the image and launch offline inference
-docker run --network host --env VLLM_CPU_KVCACHE_SPACE=1 --name cpu-test cpu-test python3 vllm/examples/offline_inference.py
+# Run the image
+docker run -itd -v ~/.cache/huggingface:/root/.cache/huggingface --network host -e HF_TOKEN --env VLLM_CPU_KVCACHE_SPACE=4 --name cpu-test cpu-test
+
+# offline inference
+docker exec cpu-test bash -c "python3 examples/offline_inference.py"
+
+# Run basic model test
+docker exec cpu-test bash -c "cd tests;
+  pip install pytest Pillow protobuf
+  bash ../.buildkite/download-images.sh
+  cd ../
+  pytest -v -s tests/models --ignore=tests/models/test_llava.py \
+    --ignore=tests/models/test_big_models.py --ignore=tests/models/test_embedding.py"
+
+# Run big model test
+docker exec cpu-test bash -c "
+  #TODO: remove this after CPU float16 support ready
+  sed -i 's/half/float/g' tests/models/test_big_models.py
```

Comment:
I think this should be inside the code. You can change `test_big_models.py` to conditionally use `float` on CPU and `half` for GPU tests, and add a comment.

Reply:
thanks, good point, fixed in https://github.com/vllm-project/vllm/pull/4113/commits/f3e300ea37e4b08912b453c5af665d1d8d1cd481